labelmerge/run.py

```diff
@@ -8,6 +8,7 @@
 def get_parser():
     """Exposes parser for sphinx doc generation, cwd is the docs dir"""
     app = SnakeBidsApp("../labelmerge", skip_parse_args=True)
+    add_dynamic_args(app.parser, app.config["parse_args"], app.config["pybids_inputs"])
```

Comment:
Need to add `from snakebids.cli import add_dynamic_args` above with the other imports for this to work